src/KDD2020-HandsOnTutorial/README.md

```diff
@@ -0,0 +1,20 @@
+## KDD 2020 hands-on tutorial
```

Comment:
Should this page exist? You're linking to src/MAG-Samples/KDD2020-HandsOnTutorial from the main README, not this page

Reply:
deleted.